neutrons/RefRed
# removed john, renamed instances of jim to plot

**Description of work.**
PR addresses the depreciation of the John plot, replacing it with just the Jim plot under a different name.

**To test:**
Run the program and load a configuration from the `test/data` directory.

In the table on the bottom right check at least of of the rows as "plotted" by clicking the checkbox in the left most column.

Data should appear on the charts on the top right, click on the right most chart to open the pop up that used to contain the Jim and John plots.  There should just be one plot now.  Test to see if it functions as before but with just the on chart.


#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
